assets/src/data/model/test/__mocks__/@eventespresso/eejs.js

```diff
@@ -0,0 +1,20 @@
+const { Exception } = require( '../../../../../eejs/index' );
+
+module.exports.data = {
+	paths: {
+		collection_endpoints: {
+			events: '/ee/v4.8.36/events',
+			tickets: '/ee/v4.8.36/tickets',
+			venues: '/ee/v4.8.36/venues',
+			terms: '/ee/v4.8.36/terms',
+		},
+		primary_keys: {
+			events: 'EVT_ID',
+			tickets: 'TKT_ID',
+			venues: 'VNU_ID',
+			terms: [ 'TERM_ID', 'TAXONOMY_ID' ],
+		},
+	},
+};
+
+module.exports.Exception = Exception;
```

Comment:
This mock allows for some nice prepped data for various tests that will rely on the data passed from the server.  Note, this is simply passing through the `Exception` function so it can be relied on as is.